### ISSUE TITLE
Can I get the original position of a marker?

### ISSUE THREAD

**Issue by t2oh4e** [jawj/OverlappingMarkerSpiderfier#26]
I implemented the plugin in a project and it works really fine.
But there is one thing I really miss: I use the markers position to calculate the distance between my location and the marker. When a marker is spiderfied it changes its position. Due to this my calculations are not correct. In most cases it's just a few meters, but sometimes several hundred meter if I have a large amount of markers and therefor a big spiral.

Long story short: Is there any way to get the markers original position without unspiderfying them?

**Comment by tverilytt** [comment]
First of all - thanks a bunch for a great utility :-)
It really is most helpful in a Google Maps mashup application I made, where potentially a lot or markers migt be on the same / near position. E.g. markers for Instagrams, Eventfuls etc. (one can check it out here: http://jo2maps.mybluemix.net/map).

I also stumbled upon this theme on markers original position vs. spiderfied position. 
I realised in in connection with using Google maps direction service to get walking directions between two markers. 
For now I will use the undocumented _omsData.l property instead of adding my own property.
